scripts/run_e2e_tests.sh

```diff
@@ -94,6 +94,7 @@ stop_mock_analytics_server() {
 password=$(parse_env_variable PASSWORD .maestro/.env.secret)
 
 if [ "$target" == "test" ]; then
+  tracking_tests_value=true
```

Comment:
j'imagine que tu n'as pas essayé de d'éxécuter `yarn test:e2e:android:staging:cloud`

à mon avis, ça ne fonctionne plus

Reply:
Effectivement ça ne fonctionnait plus, pour palier à ça j'ai conditionné l'état de cette valeur en fonction de la target en début de script, je met dans mon prochain commit si OK pour toi (marche en local et en cloud pour le coup) :
```suggestion
  case "$target" in
  "test")
    TAGS=""
    run_tracking_tests=true
    ;;

  "cloud")
    TAGS="--include-tags cloud"
    run_tracking_tests=false
    ;;
esac
```